image-layout.md

```diff
@@ -163,16 +163,9 @@ Those tags will often be represented in an image-layout repository with matching
 ```json,title=Image%20Index&mediatype=application/vnd.oci.image.index.v1%2Bjson
 {
   "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.index.v1+json",
   "manifests": [
     {
-      "mediaType": "application/vnd.oci.image.index.v1+json",
-      "size": 7143,
-      "digest": "sha256:0228f90e926ba6b96e4f39cf294b2586d38fbb5a1e385c05cd1ee40ea54fe7fd",
-      "annotations": {
-        "org.opencontainers.image.ref.name": "stable-release"
-      }
-    },
```

Comment:
@wking  I think you said conflicts with @vbatts said. He said that's a feature. If it is not, I think there is no need to fix because spec actually says ignore unknown media types